elements/rh-button-group/demo/icons.html

```diff
@@ -0,0 +1,11 @@
+<rh-button-group>
+  <rh-button variant="secondary" icon="edit" label="Edit"></rh-button>
+  <rh-button variant="secondary" icon="copy" label="Copy"></rh-button>
+  <rh-button variant="secondary" icon="undo" label="Undo"></rh-button>
+  <rh-button variant="secondary" icon="trash" label="Delete"></rh-button>
+</rh-button-group>
```

Comment:
```suggestion
<rh-button-group>
  <rh-button variant="secondary" icon="edit" accessible-label="Edit"></rh-button>
  <rh-button variant="secondary" icon="copy" accessible-label="Copy"></rh-button>
  <rh-button variant="secondary" icon="undo" accessible-label="Undo"></rh-button>
  <rh-button variant="secondary" icon="trash" accessible-label="Delete"></rh-button>
</rh-button-group>
```

We've updated this API recently and are going to deprecate `label` when `staging/eevee` releases. See #2762 for more info.